### DISCUSSION TITLE
Elsa work flow 2.0 sample

### DISCUSSION THREAD

**Discussion by satishviswanathan** [elsa-workflows/elsa-core#2674] [Q&A]
First of all thank you for the library, I'm new to this and trying to get myself started on version 2.0 +.

I was trying to follow the following sample repo but there seems to be many breaking changes. Do you have any similar example using the latest version.

https://github.com/elsa-workflows/elsa-samples

**Comment by sfmskywalker** [comment]
Hi @satishviswanathan , glad to see you've found your way over here!

The closest sample I have is [this one](https://github.com/elsa-workflows/elsa-guides/tree/master/src/ElsaDemos.DocumentManagement), which is the accompanying code for [this blog series](https://sipkeschoorstra.medium.com/building-workflow-driven-net-applications-with-elsa-2-part-1-44e08a9ba94b).

It's not the same, but it demonstrates most important concepts.

**Reply by satishviswanathan** [reply]
Thank you for your quick reply. Let me go through this one.

**Reply by satishviswanathan** [reply]
Hi @sfmskywalker I got it working. I was trying to debug the workflow and had some break points in my custom activities but it doesn't hit those. Not sure if I'm missing anything or do I need to anything else to get the debugging working.

**Reply by sfmskywalker** [reply]
You shouldn't have to do anything to get debugging working. There are different reasons why a breakpoint may not hit. The most obvious one of course being that your code is simply not executed. But assuming your activities do get executed, make sure that you launch your project in debug mode. Sometimes assemblies are "stale" and you need to do a rebuild (and sometimes even manually clean out the bin and obj folders) and try again.
